kubernetes/schema_pod_template.go

```diff
@@ -1,19 +1,21 @@
 package kubernetes
 
 import (
+	"fmt"
+
 	"github.com/hashicorp/terraform/helper/schema"
 )
 
-func podTemplateFields(isUpdatable bool) map[string]*schema.Schema {
+func podTemplateFields(controller string) map[string]*schema.Schema {
```

Comment:
Would it make senes to call this "owner" instead of "controller"?
AFAIK this type of relationship is called "ownership" in Kubernetes talk. ObjectMeta even includes an "ownerReferences" attribute.

Reply:
Done.